tequilapi/contract/connection.go

```diff
@@ -18,13 +18,73 @@
 package contract
 
 import (
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/mysteriumnetwork/node/consumer/bandwidth"
 	"github.com/mysteriumnetwork/node/core/connection"
 	"github.com/mysteriumnetwork/node/datasize"
+	"github.com/mysteriumnetwork/node/tequilapi/validation"
 	"github.com/mysteriumnetwork/payments/crypto"
 )
 
-// NewConnectionStatisticsDTO maps consumer connection details.
+var emptyAddress = common.Address{}
+
+// NewConnectionStatusDTO maps to API connection status.
+func NewConnectionStatusDTO(connection connection.Status) ConnectionStatusDTO {
```

Comment:
Can be shorter like `conn connection.Status`, it's confusing to read when your variable name is the same as package name.